README.md

```diff
@@ -14,6 +14,13 @@
 [Cloud Pub/Sub](https://cloud.google.com/pubsub/docs) is a fully-managed real-time messaging service that allows
 you to send and receive messages between independent applications.
 
+This README links to an [API reference](https://googleapis.dev/nodejs/pubsub/latest/index.html#reference), samples,
```

Comment:
Do these need to be regenerated?

Reply:
the generate now whenever we release a new package version.